themes/app/src/scss/components/_navigation.scss

```diff
@@ -48,35 +45,73 @@ $verticalGap: 15px;
 }
 
 .main-nav__main-items {
-    border-color: $menuBorderColor;
+    border-color: $sand;
     border-style: solid;
-    border-width: 0 0 $menuBorderWidth;
+    border-width: 0 0 1px;
```

Comment:
Can't the constants used previously be changed instead of hard coding them?

Reply:
Nope, there's no point since the padding are different for other elements.